syft/format/syftjson/model/package.go

```diff
@@ -32,10 +32,17 @@ type PackageBasicData struct {
 	Locations []file.Location `json:"locations"`
 	Licenses  licenses        `json:"licenses"`
 	Language  pkg.Language    `json:"language"`
-	CPEs      []string        `json:"cpes"`
+	CPEs      cpes            `json:"cpes"`
 	PURL      string          `json:"purl"`
 }
 
+type cpes []SourcedCPE
+
+type SourcedCPE struct {
```

Comment:
@wagoodman should this be renamed too?

Reply:
it would make it easy to follow

Reply:
What should the field that's currently called `CPE` be called if we rename? CPE.CPE is a little awkward. The JSON still has `cpe`.